test/Polly.Caching.Tests/HybridCacheResilienceStrategyTests.cs

```diff
@@ -0,0 +1,181 @@
+                (Func<CancellationToken, ValueTask<string>>)(static _ => new("x")),
+                CancellationToken.None);
+        });
+    }
+
+    [Fact]
+    public void Options_Defaults_Are_Documented()
+    {
+        var options = new HybridCacheStrategyOptions<string>();
+        options.Ttl.ShouldBe(TimeSpan.FromMinutes(5));
+        options.UseSlidingExpiration.ShouldBeFalse();
+    }
+
+    [Fact]
+    public async Task ValueFactory_Propagates_Callback_Exception()
+    {
+        var services = new ServiceCollection();
+        services.AddHybridCache();
+        using var provider = services.BuildServiceProvider();
+
+        var cache = provider.GetRequiredService<HybridCache>();
+
+        var options = new HybridCacheStrategyOptions<string>
+        {
+            Cache = cache,
+            CacheKeyGenerator = _ => "err-key"
+        };
+
+        var pipeline = new ResiliencePipelineBuilder<string>()
+            .AddHybridCache(options)
+            .Build();
+
+        await Should.ThrowAsync<InvalidOperationException>(async () =>
+        {
+            _ = await pipeline.ExecuteAsync(
+                (Func<CancellationToken, ValueTask<string>>)(static _ =>
+                    new(Task.FromException<string>(new InvalidOperationException("boom")))),
+                CancellationToken.None);
+        });
+    }
+
+    [Fact]
+    public async Task DefaultKeyGenerator_NullOperationKey_Throws()
+    {
+        var services = new ServiceCollection();
+        services.AddHybridCache();
+        using var provider = services.BuildServiceProvider();
+        var cache = provider.GetRequiredService<HybridCache>();
+
+        var options = new HybridCacheStrategyOptions<string>
+        {
+            Cache = cache,
+            CacheKeyGenerator = null
+        };
+
+        var pipeline = new ResiliencePipelineBuilder<string>()
+            .AddHybridCache(options)
+            .Build();
+
+        await Should.ThrowAsync<InvalidOperationException>(async () =>
+        {
+            _ = await pipeline.ExecuteAsync(
+                (Func<CancellationToken, ValueTask<string>>)(static _ => new("v")),
+                CancellationToken.None);
+        });
+    }
+
+    [Fact]
+    public void NullCache_Throws_ValidationException()
+    {
+        var options = new HybridCacheStrategyOptions<string> { Cache = null };
+
+        var builder = new ResiliencePipelineBuilder<string>();
+        Should.Throw<System.ComponentModel.DataAnnotations.ValidationException>(() =>
+            builder.AddHybridCache(options).Build());
+    }
+
+    [Fact]
+    public void StrategyCtor_NullCache_Throws_ArgumentException()
+    {
+        // Directly construct the internal strategy to bypass builder validation
+        var options = new HybridCacheStrategyOptions<string>(); // Cache is null by default
+        Should.Throw<ArgumentException>(() => new HybridCacheResilienceStrategy<string>(options));
+    }
```

Comment:
If this is impossible for a user achieve, then it's likely dead code we can remove, rather than resorting to `[InternalsVisibleTo]` just to cover the condition.